docs/src/pages/components/badge.mdx

```diff
@@ -1,2 +1,231 @@
+import { Link } from 'gatsby';
+import { BadgeWithParentExample } from '../../components/BadgeWithParentExample';
+
 # Badge
 
+The Badge component displays a dot or a circle with a count. It is normally used to indicate pending items (items in cart, unread messages, etc.)
+
+## Usage
+
+There are three sizes: `dot` (default) and `small` are meant to be used with an icon (Anchor comes with <Link to="/components/icon">its own set of icons</Link>). `large` is meant to be used standalone.
```

Comment:
Bold **Anchor** whenever you use the word. When you link, only link the word of the component, so in this case, `Icon`.